docs/Drawer.md

```diff
@@ -62,20 +62,20 @@ const xsDown = useMediaQuery(theme.breakpoints.down('xs'));
+| ------------------------------- | ------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------ | -------- | ------------ |
+| activeItem                      | itemID for the 'active' item                                                                            | `string`                                                     | no       |              |
+| classes                         | Style overrides                                                                                         | `DrawerClasses`                                              | no       |              |
+| condensed                       | Enables a condensed view for the `rail` variant which removes NavItem labels and shows tooltips instead | `boolean`                                                    | no       | `false`      |
+| noLayout                        | Set to true if used without a `<DrawerLayout>`                                                          | `boolean`                                                    | no       | `false`      |
+| onItemSelect                    | A callback function to execute whenever an item is clicked                                              | `(id: string) => void`                                       | no       |              |
+| open                            | Controls the open/closed state of the drawer                                                            | `boolean`                                                    | yes      |              |
+| openOnHover                     | Automatically open the drawer on hover when closed (persistent variant only)                            | `boolean`                                                    | no       | `true`       |
+| openOnHoverDelay                | Delay (ms) before triggering open on hover (persistent variant only)                                    | `number`                                                     | no       | `500`        |
+| sideBorder                      | Whether to use a side border for the drawer instead of a shadow                                         | `boolean`                                                    | no       | `false`      |
+| variant                         | The variant to use (see below)                                                                          | `'permanent'` \| `'persistent'` \| `'temporary'` \| `'rail'` | no       | `persistent` |
+| width                           | Sets the width of the drawer (in px) when open                                                          | `number` \| `string`                                         | no       | `22.5rem`    |
```

Comment:
Just curious if we have been listing default values in rems (technically correct) or if we have been using the px value (easier for people to understand) with the assumption of 1 rem = 16px. Or maybe we show both like you did in the intellisense.